src/app/store/users/users.actions.ts

```diff
@@ -1,3 +1,6 @@
-import { createAction } from '@ngrx/store';
+import { createAction, props } from '@ngrx/store';
+import { User } from '../../models/user';
 
-export const myAction = createAction('[User] my action');
+export const loadUsers = createAction('[User List] Load Users');
+export const loadUsersSuccess = createAction('[User List] Load Users Success', props<{ payload: User[] }>());
```

Comment:
payload - users

Reply:
jo mel jsem to jako user, ale prislo me ze se clovek pak ztrati prehled co je co a odkud :D ale to je asi zvykem prijde